cwltool/argparser.py

```diff
@@ -418,7 +418,7 @@ def add_argument(toolparser, name, inptype, records, description="",
                 toolparser, fieldname, fieldtype, records,
                 fielddescription)
         return
-    if inptype == "string":
+    elif inptype == "string":
```

Comment:
no need for an `elif` after a `return`

Reply:
This would be true if all the cases before this one would end the function.

If I don't put the `elif`, the last `else` returns `None` when `inptype` is `'File'`,  `'Directory'`, etc